src/Components/Authentication/Authentication.js

```diff
@@ -7,16 +7,19 @@ import Unavailable from '@redhat-cloud-services/frontend-components/Unavailable'
 import propTypes from 'prop-types';
 import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
 import NotAuthorized from '@redhat-cloud-services/frontend-components/NotAuthorized';
+import { usePermissionsWithContext } from '@redhat-cloud-services/frontend-components-utilities/RBACHook';
 
 const Authentication = ({ children }) => {
   const queryClient = useQueryClient();
   const location = useLocation();
   const chrome = useChrome();
-  const { isLoading, isFetching, isSuccess, isError, data } = useUser();
-  const hasAnyPermission =
-    data?.rbacPermissions &&
-    (data.rbacPermissions.canReadActivationKeys ||
-      data.rbacPermissions.canWriteActivationKeys);
+  const { isLoading, isFetching, isSuccess, isError } = useUser();
+  const { hasAccess: hasReadPermission } = usePermissionsWithContext([
+    'config-manager:activation_keys:read',
+  ]);
+  const { hasAccess: hasWritePermission } = usePermissionsWithContext([
+    'config-manager:activation_keys:write',
+  ]);
```

Comment:
Can't we skip the `useUser` hook here?

Reply:
`isLoading`, `isFetching`, `isSuccess`, `isError` are used below

Reply:
Oh yeah, I see now, you're right.